<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminSystemWidget</class>
 <widget class="QWidget" name="AdminSystemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border: none;
}
#header{
	background-color: #d0e6ff;
}
#body{
	background-color: #f5faf5;
}
#footer{
	background-color: #d0e6ff;
}
QMessageBox {
    background-color: #f5faf5;
    border: 2px solid #020f13;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QFrame" name="header">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="group_bt_manage">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: transparent;
	color: dimgray;
	padding: 10px;
	padding-left: 20px;
	border-radius: 0px;
	text-align: left
}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color: #a8dadc;

}


QPushButton::pressed
{
	background-color: #83c5be;

}
QPushButton::checked
{
	background-color: #4cb1ff;
	color: #020f13;
}</string>
        </property>
        <layout class="QHBoxLayout" name="group_func_bt" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bt_manage_device">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>  Quản lý thiết bị</string>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/assets/icons/camera.svg</normaloff>:/assets/icons/camera.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_manage_user">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>  Quản lý danh sách</string>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/assets/icons/user-check.svg</normaloff>:/assets/icons/user-check.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="body">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QStackedWidget" name="page_manage">
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_manage_device">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableView" name="tb_device_list">
            <property name="styleSheet">
             <string notr="true">QTableView {
    color: black;
    gridline-color: black;
    border-color: #ffafcc;
    font: 1.2em;
}
QHeaderView::section {
    background-color: #219ebc;
    color: #000814;
    height: 35%;
    font: 1.5em;
}

QTableView::item:selected {
    background-color: #bde0fe;
	color: black;
	border: 1px solid #023047;
}s

QScrollBar:vertical {
    background: #bde0fe;
}
 QScrollBar::handle:vertical {
    background: #a2d2ff;
 }
QScrollBar:horizontal {
    background: #bde0fe;
}
 QScrollBar::handle:horizontal {
    background: #a2d2ff;
 }</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_manage_user">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableView" name="tb_user_list">
            <property name="styleSheet">
             <string notr="true">QTableView {
    color: black;
    gridline-color: black;
    border-color: #ffafcc;
    font: 1.2em;
}
QHeaderView::section {
    background-color: #219ebc;
    color: #000814;
    height: 35%;
    font: 1.5em;
}

QTableView::item:selected {
    background-color: #bde0fe;
	color: black;
	border: 1px solid #023047;
}s

QScrollBar:vertical {
    background: #bde0fe;
}
 QScrollBar::handle:vertical {
    background: #a2d2ff;
 }
QScrollBar:horizontal {
    background: #bde0fe;
}
 QScrollBar::handle:horizontal {
    background: #a2d2ff;
 }</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QFrame" name="footer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>522</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_add_user">
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: #76c893;
	color: #020f13;
	padding: 10px;
	border-radius: 20px;
	text-align: center
}

QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color: #99d98c;

}


QPushButton::pressed
{
	background-color: #52b69a;

}</string>
        </property>
        <property name="text">
         <string>  Add</string>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/assets/icons/ic_add_user.svg</normaloff>:/assets/icons/ic_add_user.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
